swap_meet/item.py

```diff
@@ -1,2 +1,25 @@
 class Item:
-    pass
+   def __init__(self, category = "", condition = 0):
+      
+       self.category = category
+       self.condition = condition 
+   
+   def __str__(self):
+       return "Hello World!"
+
+   def condition_description(self):
+        if self.condition == 0:
+            return "Condition unknown"
+        elif 0 < self.condition <= 1:
+            return "heavily used item"
+        elif 1 < self.condition <= 2:
```

Comment:
This work and is clear! Note that it will also work if you leave off the left hand side of the conditional expression `self.condition <= 2:`, etc.